src/app/organization/organization.component.html

```diff
@@ -49,7 +49,7 @@ <h4 matLine>Standing Instructions History</h4>
             </mat-list-item>
           </div>
 
-          <div #fundmapping>
+          <div #fundmapping *ngIf="!shouldShowFundMapping">
```

Comment:
@Onyx2406 This variable name and condition seems wrong to me:
- if the `shouldShowFundMapping` variable is `true` it should show the fund mapping, but here it doing exactly the opposite.

What do you think?

Reply:
@adamsaghy right, I have fixed it now.